tictactoe.py

```diff
@@ -1,26 +1,63 @@
+empty_squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+playerX_moves = []
+player0_moves = []
+winningcombo_1 = [1, 2, 3]
+winningcombo_2 = [4, 5, 6]
+winningcombo_3 = [7, 8, 9]
+winningcombo_4 = [1, 4, 7]
+winningcombo_5 = [2, 5, 8]
+winningcombo_6 = [3, 6, 9]
+winningcombo_7 = [1, 5, 9]
+winningcombo_8 = [3, 5, 7]
+
+print("""
+Let’s play tic-tac-toe! The developer is a newbie python coder 🐍
+so for now, plz only type integers from the empty list. 😜
+""")
+
 def game():
-    print("Lets play! Only integers between 1-9 plz:")
-
-    for playermove in range(0,9):
-        playermove = int(input(">>> "))
-
-        if playermove == 1:
-            print("Top left corner, all right. ↖️")
-        if playermove == 2:
-            print("Top center. ⬆️")
-        if playermove == 3:
-            print("Top right corner, nice. ↗️")
-        if playermove == 4:
-            print("Left center. ⬅️")
-        if playermove == 5:
-            print("Yeah that’s a good one. 🆒")
-        if playermove == 6:
-            print("Right center. ➡️")
-        if playermove == 7:
-            print("Bottom left corner, good job. ↙️")
-        if playermove == 8:
-            print("Bottom center. ⬇️")
-        if playermove == 9:
-            print("Bottom right corner, cool. ↘️")
+
+    if len(empty_squares) > 1:
+
+        next_turn_is_x = True
```

Comment:
General design/philosophy remark incoming. In tic-tac toe, player X always starts and then follows player O. So in theory, we could say that the game goes like this:

1. player X puts a mark 
2. player O puts a mark
3. player X puts a mark
4. player O puts a mark
5. player X puts a mark
6. player O puts a mark
7. player X puts a mark
8. player O puts a mark
9. player X puts a mark

And in between, we need to check if anyone has won. So we don't really need to "check" who has the turn according to a `True`/`False`. Or even check if we are out of empty squares. Suppose we had a function named `turn` that did accepted a player as a parameter:

- put down a mark for that player
- if that player had won, ended the game 

Then we could code the entire tic-tac-toe like this:

1. `turn(X)`
2. `turn(O)`
3. `turn(X)`
...

Or we could build a `game_round()` function that did this:

1. turn(X)
2. turn(Y)

Then we could use `game_round()` 4 times. And then we know that there is only going to be 1 empty square after that, because we would have put down 8 marks. So we could fill in the last one after 4 `game_round()`s and check if `X` won. And then we don't actually need check whose turn it is or whether we are out of squares either. We would actually be left with only 1 single `if` in the whole program and that would be the `if` that checks if someone has won! I think it's usually worth considering to design programs so that they contain a minimum amount of `if` checks, when it is possible. They're easier on the brain to read :-)